### DISCUSSION TITLE
Toggle to Auto delete feed

### DISCUSSION THREAD

**Discussion by Bertklq1** [jocmp/capyreader#238] [Feature Requests]
### Steps to Reproduce

Sertings*

### Expected behavior

Make a toogle to Not delete the feed automatically after 3 months

**Comment by jocmp** [comment]
@Bertklq1 makes sense. I'll add this to the next release.

**Comment by jocmp** [comment]
This feature is available as of [2024.08.1028-dev](https://github.com/jocmp/capyreader/releases/tag/2024.08.1028-dev).
